packages/kitchen-sink/src/server.ts

```diff
@@ -0,0 +1,12 @@
+export * from '@rise-tools/kit/server'
+export * from '@rise-tools/kit-expo-router/server'
+export * from '@rise-tools/kit-forms/server'
+export * from '@rise-tools/kit-haptics/server'
+export * from '@rise-tools/kit-linking/server'
+export * from '@rise-tools/kit-lucide-icons/server'
+export * from '@rise-tools/kit-qrcode/server'
+export * from '@rise-tools/kit-tamagui-toast/server'
+export * from '@rise-tools/tamagui/server'
+
+// tbd: "Circle" collides with Tamagui.Circle. Rename one of them or namespace?
+export * as SVG from '@rise-tools/kit-svg/server'
```

Comment:
Should we rename "Circle" inside SVG, "Circle" inside Tamagui, or leave it exported like this? (Kinda sucks to destruct on the client, b/c you have to write const { Circle } = SVG.

Reply:
ooh damn, maybe they should both be accessed through the namespace, otherwise people may accidentally import the wrong thing.